### PR TITLE
test(@wallet): edit/delete saved addresses

### PR DESCRIPTION
**Pull request by alaibe**

### CODE REVIEW

File: test/ui-test/src/screens/StatusWalletScreen.py [hunk 1]
```diff
@@ -186,14 +189,51 @@ def add_saved_address(self, name: str, address: str):
...
+        
     def verify_saved_address_exists(self, name: str):
         list = get_obj(SavedAddressesScreen.SAVED_ADDRESSES_LIST.value)
         for index in range(list.count):
             if list.itemAtIndex(index).objectName == name:
                 return  
     
         assert False, "no saved address found"
+        
+    def verify_saved_address_dont_exists(self, name: str):
```
Comment (jrainville) [inline test/ui-test/src/screens/StatusWalletScreen.py:232]:
```suggestion
    def verify_saved_address_doesnt_exist(self, name: str):
```

File: test/ui-test/testSuites/suite_status/shared/steps/walletSteps.py [hunk 2]
```diff
@@ -46,4 +54,7 @@ def step(context):
 @Then("the name |any| is in the list of saved addresses")
 def step(context, name: str):
     _walletScreen.verify_saved_address_exists(name) 
-    
\ No newline at end of file
+    
+@Then("the name |any| is not in the list of saved addresses")
+def step(context, name: str):
+    _walletScreen.verify_saved_address_dont_exists(name)
```
Comment (jrainville) [inline test/ui-test/testSuites/suite_status/shared/steps/walletSteps.py:60]:
```suggestion
    _walletScreen.verify_saved_address_doesnt_exist(name)     
```

File: test/ui-test/testSuites/suite_status/tst_wallet/test.feature [hunk 3]
```diff
@@ -58,6 +58,24 @@ Feature: Status Desktop Wallet
         When the user adds a saved address name <name> and address <address>
         Then the name <name> is in the list of saved addresses
 
+    	Examples:
+      	  | name | address 								      |
+      	  | one  | 0x8397bc3c5a60a1883174f722403d63a8833312b7 |
+
+    Scenario Outline: User can edit a saved address
+        When the user adds a saved address name <name> and address <address>
```
Comment (jrainville) [inline test/ui-test/testSuites/suite_status/tst_wallet/test.feature:66]:
```suggestion
        When the user adds a saved address named <name> and with address <address>
```

File: test/ui-test/testSuites/suite_status/tst_wallet/test.feature [hunk 4]
```diff
@@ -58,6 +58,24 @@ Feature: Status Desktop Wallet
...
+        And the user edits a saved address with name <name> to <new_name>
+        Then the name <new_name><name> is in the list of saved addresses
```
Comment (jrainville) [inline test/ui-test/testSuites/suite_status/tst_wallet/test.feature:68]: Is this a mistake (double name)?

File: test/ui-test/testSuites/suite_status/tst_wallet/test.feature [hunk 5]
```diff
@@ -58,6 +58,24 @@ Feature: Status Desktop Wallet
...
+
+    	Examples:
+      	  | name | address 								      | new_name |
+      	  | bar  | 0x8397bc3c5a60a1883174f722403d63a8833312b7 | foo      |
+
+    Scenario Outline: User can delete a saved address
+        When the user adds a saved address name <name> and address <address>
```
Comment (jrainville) [inline test/ui-test/testSuites/suite_status/tst_wallet/test.feature:75]:
```suggestion
        When the user adds a saved address named <name> and with address <address>
```

**Review by jrainville — APPROVED** [review]
Good job. I only found typos.

You'll need to rebase (the base branch) to have the cleaned up names.py

File: test/ui-test/testSuites/suite_status/tst_wallet/test.feature (hunk 4 above)
Comment (alaibe) [reply test/ui-test/testSuites/suite_status/tst_wallet/test.feature:68]: This is because the input preprend by default.

**Pull request merged**